api/src/main/java/com/cloud/template/VirtualMachineTemplate.java

```diff
@@ -73,6 +73,8 @@
         sharedexecutable, // ready templates that have been granted to the calling user by another user
         executable, // templates that are owned by the calling user, or public templates, that can be used to deploy a
         community, // returns templates that have been marked as public but not featured
+        system, // system vm templates
+        inactive, // inactive system vm templates
         all // all templates (only usable by admins)
```

Comment:
Can you explain why we're adding new template filters?

Reply:
The "system" filter is used to get the system template id for seeding.
The "inactive" filter is used to list all inactive system vm templates. The list is used on the front end to list other system vm templates that are not currently used. We needed this view because the "all" filter doesn't show inactive system vm templates.